crates/store/re_types/src/components/draw_order_ext.rs

```diff
@@ -8,6 +8,9 @@ impl DrawOrder {
     /// Draw order used for depth image when they're not shown as a 3D point cloud and no draw order was specified.
     pub const DEFAULT_DEPTH_IMAGE: Self = Self(Float32(-20.0));
 
+    /// Draw order used for images if no draw order was specified.
+    pub const DEFAULT_VIDEO_FRAME_REFERENCE: Self = Self(Float32(-15.0));
```

Comment:
figured between depth and images makes sense - it's still like an image but if there is also an image it should show on top
🤷